content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -35,14 +35,21 @@ components still rely on Endpoints. For now, enabling Endpoint Slices should be
 seen as an addition to Endpoints in a cluster, not a replacement for them.
 {{< /note >}}
 
-As an alpha feature, Endpoint Slices are not enabled by default in Kubernetes.
-To enable them, the EndpointSlice feature gate will need to be enabled
-(`--feature-gates=EndpointSlice=true`).
+Endpoint Slices are considered a beta feature, but only the API is enabled by
```

Comment:
I'd move this near to the top of the page, so that it's close to the feature-state shortcode.

Reply:
I'm not sure how I should be structuring this. Above this there's an intro, a feature-state, and then a note. Agree that it would be nice to make both this and the feature state more prominent, but I'm not sure how to restructure/reorder this.